# Portuguese (Brazil) translations for zaqar.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the zaqar project.
#
# Translators:
# Gabriel Guerreiro <gabriel.eu.br@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version:  Zaqar\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-09-19 06:37+0000\n"
"PO-Revision-Date: 2014-09-16 09:26+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Portuguese (Brazil) "
"(http://www.transifex.com/projects/p/zaqar/language/pt_BR/)\n"
"Plural-Forms: nplurals=2; plural=(n > 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: zaqar/common/api.py:49
msgid "{0} is not a valid operation"
msgstr "{0} não é uma operação válida"

#: zaqar/common/cli.py:50
msgid "Terminating"
msgstr "Finalizando"

#: zaqar/common/pipeline.py:94
#, python-format
msgid "Stage %(stage)s does not implement %(method)s"
msgstr "O estágio %(stage)s não implementa o método %(method)s"

#: zaqar/common/pipeline.py:108
#, python-format
msgid "Method %s not found in any of the registered stages"
msgstr "O método %s não foi encontrado em nenhum dos estágios registrados"

#: zaqar/common/transport/wsgi/helpers.py:44
msgid "Malformed hexadecimal UUID."
msgstr "UUID hexadecimal inválido."

#: zaqar/common/transport/wsgi/helpers.py:64
msgid ""
"\n"
"X-PROJECT-ID cannot be an empty string. Specify the right header X"
"-PROJECT-ID\n"
"and retry."
msgstr ""
"\n"
"X-PROJECT-ID não pode ser uma string vazia. Especifique o cabeçalho X"
"-PROJECT-ID\n"
"e tente novemente."

#: zaqar/common/transport/wsgi/helpers.py:71
msgid "The header X-PROJECT-ID was missing"
msgstr ""

#: zaqar/common/transport/wsgi/helpers.py:129
msgid "Invalid queue identification"
msgstr "Identificação de fila inválida"

#: zaqar/common/transport/wsgi/helpers.py:130
msgid "The format of the submitted queue name or project id is not valid."
msgstr ""
"O formato do nome da fila submetida ou o identificador do projeto não é "
"válido."

#: zaqar/openstack/common/gettextutils.py:301
msgid "Message objects do not support addition."
msgstr "Objetos mensagem não suportam adição."

#: zaqar/openstack/common/gettextutils.py:311
msgid ""
"Message objects do not support str() because they may contain non-ascii "
"characters. Please use unicode() or translate() instead."
msgstr ""
"Objetos mensagem não suportam str() pois podem conter carácteres "
"não-ascii. Use unicode() ou translate()."

#: zaqar/openstack/common/lockutils.py:101
#, python-format
msgid "Unable to acquire lock on `%(filename)s` due to %(exception)s"
msgstr "Não foi possível bloquear o arquivo ` %(filename)s` devido a %(exception)s"

#: zaqar/openstack/common/log.py:287
#, python-format
msgid "Deprecated: %s"
msgstr "Obsoleto: %s"

#: zaqar/openstack/common/log.py:396
#, python-format
msgid "Error loading logging config %(log_config)s: %(err_msg)s"
msgstr "Erro ao carregar configuração %(log_config)s: %(err_msg)s"

#: zaqar/openstack/common/log.py:457
#, python-format
msgid "syslog facility must be one of: %s"
msgstr "O recurso de syslog deve ser um dos seguintes: %s"

#: zaqar/openstack/common/log.py:700
#, python-format
msgid "Fatal call to deprecated config: %(msg)s"
msgstr "Chamada fatal de configuração obsoleta: %(msg)s"

#: zaqar/openstack/common/strutils.py:114
#, python-format
msgid "Unrecognized value '%(val)s', acceptable values are: %(acceptable)s"
msgstr "Valor não reconhecido '%(val)s', os valores aceitáveis são: %(acceptable)s"

#: zaqar/openstack/common/strutils.py:219
#, python-format
msgid "Invalid unit system: \"%s\""
msgstr "Unidade inválida do sistema: \"%s\""

#: zaqar/openstack/common/strutils.py:228
#, python-format
msgid "Invalid string format: %s"
msgstr "Formato de string inválido: %s"

#: zaqar/queues/storage/pipeline.py:30
msgid ""
"Pipeline to use for processing {0} operations. This pipeline will be "
"consumed before calling the storage driver's controller methods."
msgstr ""

#: zaqar/queues/storage/pipeline.py:77
#, python-format
msgid "Stage %(stage)d could not be imported: %(ex)s"
msgstr "O estágio %(stage)d não pode ser importado: %(ex)s"

#: zaqar/queues/storage/mongodb/driver.py:87
#, python-format
msgid "The mongodb driver requires mongodb>=2.2,  %s found"
msgstr ""

#: zaqar/queues/storage/mongodb/messages.py:547
#, python-format
msgid ""
"%(attempts)d attempt(s) required to post %(num_messages)d messages to "
"queue \"%(queue)s\" under project %(project)s"
msgstr ""
"%(attempts)d tentativa(s) necessária(s) para enviar %(num_messages)d "
"mensagens para a fila \"%(queue)s\" do projeto %(project)s"

#: zaqar/queues/storage/mongodb/messages.py:591
#, python-format
msgid ""
"First attempt failed while adding messages to queue \"%(queue)s\" under "
"project %(project)s"
msgstr ""
"Primeira tentativa falhou durante a adição de mensagens à fila "
"\"%(queue)s\" do projeto %(project)s"

#: zaqar/queues/storage/mongodb/messages.py:605
#, python-format
msgid ""
"Exceeded maximum retry duration for queue \"%(queue)s\" under project "
"%(project)s"
msgstr ""
"Excedido o tempo máximo de tentativas para a fila \"%(queue)s\" do "
"projeto %(project)s"

#: zaqar/queues/storage/mongodb/messages.py:665
#, python-format
msgid ""
"Hit maximum number of attempts (%(max)s) for queue \"%(queue)s\" under "
"project %(project)s"
msgstr ""
"Excedido o número máximo de tentativas (%(max)s) para a fila "
"\"%(queue)s\" do projeto %(project)s"

#: zaqar/queues/storage/mongodb/queues.py:171
#, python-format
msgid ""
"Failed to increment the message counter for queue %(name)s and project "
"%(project)s"
msgstr ""
"Falha ao incrementar o contador de mensagens para a fila %(name)s e "
"projeto %(project)s"

#: zaqar/queues/storage/mongodb/utils.py:290
msgid "Caught AutoReconnect, retrying the call to {0}"
msgstr "Exceção AutoReconnect capturada, tentando chamar {0} novamente"

#: zaqar/queues/storage/mongodb/utils.py:296
msgid "Caught AutoReconnect, maximum attempts to {0} exceeded."
msgstr ""
"Exceção AutoReconnect capturada, número máximo de tentativas para {0} "
"excedido."

#: zaqar/queues/storage/redis/utils.py:157
msgid "Caught ConnectionError, retrying the call to {0}"
msgstr ""

#: zaqar/queues/storage/redis/utils.py:162
msgid "Caught ConnectionError, maximum attempts to {0} exceeded."
msgstr ""

#: zaqar/queues/storage/sqlalchemy/driver.py:42
msgid ""
"sqlalchemy's data plane driver will be removed during the next release. "
"Please, consider moving your data to one of the other supported drivers."
msgstr ""

#: zaqar/queues/transport/validation.py:97
msgid "Project ids may not be more than {0} characters long."
msgstr "Identificadores de projeto não devem ter mais de {0} carácteres."

#: zaqar/queues/transport/validation.py:101
msgid "Queue names may not be more than {0} characters long."
msgstr "Nomes de filas não devem ter mais de {0} carácteres."

#: zaqar/queues/transport/validation.py:106
msgid ""
"Queue names may only contain ASCII letters, digits, underscores, and "
"dashes."
msgstr ""
"O nome de uma fila deve conter apenas letras ASCII, números, subtraços e "
"traços."

#: zaqar/queues/transport/validation.py:119
msgid "Limit must be at least 1 and no greater than {0}."
msgstr "O limite deve ser no mínimo 1 e não deve ser superior a {0}."

#: zaqar/queues/transport/validation.py:131
msgid "Queue metadata is too large. Max size: {0}"
msgstr "Os métadados da fila são muito grandes. Tamanho máximo: {0}"

#: zaqar/queues/transport/validation.py:143
msgid "No messages to enqueu."
msgstr "Não foi possível adicionar mensagens à fila."

#: zaqar/queues/transport/validation.py:158
msgid "Message collection size is too large. Max size {0}"
msgstr "O tamanho da coleção de mensagens é muito grande. O tamnho máximo é {0}"

#: zaqar/queues/transport/validation.py:167
msgid ""
"The TTL for a message may not exceed {0} seconds, and must be at least "
"{1} seconds long."
msgstr ""
"O tempo de vida de uma mensagem não pode exceder {0} segundos e não deve "
"ser menor que {1} segundos."

#: zaqar/queues/transport/validation.py:183
#: zaqar/queues/transport/validation.py:239
msgid "Limit must be at least 1 and may not be greater than {0}."
msgstr "O limite deve ser de no mínimo 1 e não deve exceder {0}."

#: zaqar/queues/transport/validation.py:201
msgid "pop and id params cannot be present together in the delete request."
msgstr ""

#: zaqar/queues/transport/validation.py:207
msgid ""
"The request should have either \"ids\" or \"pop\" parameter in the "
"request, to be able to delete."
msgstr ""

#: zaqar/queues/transport/validation.py:214
msgid "Pop value must be at least 1 and may not be greater than {0}."
msgstr ""

#: zaqar/queues/transport/validation.py:221
msgid "ids parameter should have at least 1 and not greater than {0} values."
msgstr ""

#: zaqar/queues/transport/validation.py:248
msgid ""
"The grace for a claim may not exceed {0} seconds, and must be at least "
"{1} seconds long."
msgstr ""
"O prazo de carência para um pedido não deve exceder {0} segundos e não "
"deve ser menor que {1} segundo."

#: zaqar/queues/transport/validation.py:264
msgid ""
"The TTL for a claim may not exceed {0} seconds, and must be at least {1} "
"seconds long."
msgstr ""
"O tempo de vida de um pedido não pode exceder {0} segundos e não deve ser"
" menor que {1} segundos."

#: zaqar/queues/transport/wsgi/driver.py:106
#, python-format
msgid "Serving on host %(bind)s:%(port)s"
msgstr "Servindo em %(bind)s:%(port)s"

#: zaqar/queues/transport/wsgi/errors.py:24
msgid "Service temporarily unavailable"
msgstr "Serviço temporariamente indisponível"

#: zaqar/queues/transport/wsgi/errors.py:25
msgid "Please try again in a few seconds."
msgstr "Tente novamente em alguns segundos."

#: zaqar/queues/transport/wsgi/errors.py:36
msgid "Invalid API request"
msgstr ""

#: zaqar/queues/transport/wsgi/errors.py:45
msgid "Invalid request body"
msgstr "Corpo da requisição inválido"

#: zaqar/queues/transport/wsgi/errors.py:54
msgid "Document type not supported."
msgstr "Tipo de documento não suportado."

#: zaqar/queues/transport/wsgi/utils.py:50
msgid "Request body can not be empty"
msgstr "O corpo da requisição não pode ser vazio"

#: zaqar/queues/transport/wsgi/utils.py:61
msgid "Request body could not be parsed."
msgstr "Não foi possível analisar o corpo da requisição."

#: zaqar/queues/transport/wsgi/utils.py:66
msgid "JSON contains integer that is too large."
msgstr "JSON possui um numero inteiro muito grande."

#: zaqar/queues/transport/wsgi/utils.py:72
msgid "Request body could not be read."
msgstr "O corpo da requisição não pode ser lido."

#: zaqar/queues/transport/wsgi/utils.py:168
msgid "Missing \"{name}\" field."
msgstr "Faltando o campo \"{name}\"."

#: zaqar/queues/transport/wsgi/utils.py:176
msgid "The value of the \"{name}\" field must be a {vtype}."
msgstr "O valor do campo \"{name}\" deve ser do tipo {vtype}."

#: zaqar/queues/transport/wsgi/v1_0/claims.py:77
#: zaqar/queues/transport/wsgi/v1_1/claims.py:100
msgid "Claim could not be created."
msgstr "Não foi possível criar o pedido."

#: zaqar/queues/transport/wsgi/v1_0/claims.py:125
#: zaqar/queues/transport/wsgi/v1_1/claims.py:148
msgid "Claim could not be queried."
msgstr "Não foi possível consultar o pedido."

#: zaqar/queues/transport/wsgi/v1_0/claims.py:173
#: zaqar/queues/transport/wsgi/v1_1/claims.py:195
msgid "Claim could not be updated."
msgstr "Não foi possível atualizar o pedido."

#: zaqar/queues/transport/wsgi/v1_0/claims.py:191
#: zaqar/queues/transport/wsgi/v1_1/claims.py:213
msgid "Claim could not be deleted."
msgstr "Não foi possível excluir o pedido."

#: zaqar/queues/transport/wsgi/v1_0/messages.py:61
#: zaqar/queues/transport/wsgi/v1_0/messages.py:261
#: zaqar/queues/transport/wsgi/v1_1/messages.py:74
#: zaqar/queues/transport/wsgi/v1_1/messages.py:320
msgid "Message could not be retrieved."
msgstr "Não foi possível recuperar a mensagem."

#: zaqar/queues/transport/wsgi/v1_0/messages.py:104
#: zaqar/queues/transport/wsgi/v1_1/messages.py:119
msgid "Messages could not be listed."
msgstr "Não foi possível listar a mensagem."

#: zaqar/queues/transport/wsgi/v1_0/messages.py:167
#: zaqar/queues/transport/wsgi/v1_1/messages.py:192
msgid "No messages could be enqueued."
msgstr "Nenhuma mensagem pode ser adicionada à fila."

#: zaqar/queues/transport/wsgi/v1_0/messages.py:172
#: zaqar/queues/transport/wsgi/v1_1/messages.py:197
msgid "Messages could not be enqueued."
msgstr "Não foi possível adicionar mensagens à fila."

#: zaqar/queues/transport/wsgi/v1_0/messages.py:230
#: zaqar/queues/transport/wsgi/v1_1/messages.py:265
msgid "Messages could not be deleted."
msgstr "Não foi possível excluir as mensagens."

#: zaqar/queues/transport/wsgi/v1_0/messages.py:277
#: zaqar/queues/transport/wsgi/v1_1/messages.py:338
#: zaqar/queues/transport/wsgi/v1_1/pools.py:178
msgid "Unable to delete"
msgstr "Impossível excluir"

#: zaqar/queues/transport/wsgi/v1_0/messages.py:288
#: zaqar/queues/transport/wsgi/v1_1/messages.py:349
msgid "A claim was specified, but the message is not currently claimed."
msgstr ""

#: zaqar/queues/transport/wsgi/v1_0/messages.py:294
#: zaqar/queues/transport/wsgi/v1_1/messages.py:355
msgid "The specified claim does not exist or has expired."
msgstr ""

#: zaqar/queues/transport/wsgi/v1_0/messages.py:300
#: zaqar/queues/transport/wsgi/v1_1/messages.py:361
msgid "This message is claimed; it cannot be deleted without a valid claim ID."
msgstr ""

#: zaqar/queues/transport/wsgi/v1_0/messages.py:306
#: zaqar/queues/transport/wsgi/v1_1/messages.py:367
msgid "Message could not be deleted."
msgstr "A mensagem não pode ser excluída."

#: zaqar/queues/transport/wsgi/v1_0/metadata.py:54
#: zaqar/queues/transport/wsgi/v1_1/queues.py:55
msgid "Queue metadata could not be retrieved."
msgstr "Não foi possível obter os metadados da fila."

#: zaqar/queues/transport/wsgi/v1_0/metadata.py:91
msgid "Metadata could not be updated."
msgstr "Não foi possível atualizar os metadados."

#: zaqar/queues/transport/wsgi/v1_0/queues.py:48
#: zaqar/queues/transport/wsgi/v1_1/queues.py:90
msgid "Queue could not be created."
msgstr "Não foi possível criar a fila."

#: zaqar/queues/transport/wsgi/v1_0/queues.py:77
#: zaqar/queues/transport/wsgi/v1_1/queues.py:105
msgid "Queue could not be deleted."
msgstr "Não foi possível excluir a fila."

#: zaqar/queues/transport/wsgi/v1_0/queues.py:113
#: zaqar/queues/transport/wsgi/v1_1/queues.py:141
msgid "Queues could not be listed."
msgstr "Não foi possível listar as filas."

#: zaqar/queues/transport/wsgi/v1_0/stats.py:63
#: zaqar/queues/transport/wsgi/v1_1/stats.py:72
msgid "Queue stats could not be read."
msgstr "Não foi possível ler as estatísticas da fila."

#: zaqar/queues/transport/wsgi/v1_1/flavors.py:143
msgid "Flavor {flavor} could not be created. Pool {pool} does not exist"
msgstr ""

#: zaqar/queues/transport/wsgi/v1_1/flavors.py:146
msgid "Unable to create"
msgstr ""

#: zaqar/queues/transport/wsgi/v1_1/health.py:38
msgid "Health status could not be read."
msgstr ""

#: zaqar/queues/transport/wsgi/v1_1/messages.py:163
msgid "No messages were found in the request body."
msgstr ""

#: zaqar/queues/transport/wsgi/v1_1/messages.py:283
msgid "Messages could not be popped."
msgstr ""

#: zaqar/queues/transport/wsgi/v1_1/pools.py:179
msgid "This pool is used by flavors {flavor}; It cannot be deleted."
msgstr ""

